gar/sssp/.gar_config.yaml

```diff
@@ -0,0 +1,11 @@
+components:
```

Comment:
thanks neng, however, the format is outdated. maybe leave the gar related to dongze.

Reply:
I'll remove this part .